### DISCUSSION TITLE
How can I import tokens?

### DISCUSSION THREAD

**Discussion by jay-itl** [microsoft/fluentui#32961] [Q&A]
I found `fontFamilyBase` isn't setup by using 

```
<FluentProvider className="h-full">
                        <ThemeProvider theme={theme} className="h-full">
                            {children}
                        </ThemeProvider>
                    </FluentProvider>
```

Anything I missed?

![image](https://github.com/user-attachments/assets/0f3d97a5-105f-4f2c-bf4c-28a41f03134d)

Thanks,

**Comment by layershifter** [comment]
@jay-itl you need apply (pass) a `theme` to `FluentProvider` i.e `<FluentProvider theme={teamsLightTheme} />`.
